controllers/toolchaincluster/healthchecker_test.go

```diff
@@ -39,139 +34,51 @@ func TestClusterHealthChecks(t *testing.T) {
 		Reply(404)
 
 	tests := map[string]struct {
-		tctype            string
-		apiendpoint       string
-		clusterconditions []toolchainv1alpha1.Condition
-		status            toolchainv1alpha1.ToolchainClusterStatus
+		tctype      string
+		apiendpoint string
+		healthcheck bool
+		errh        error
```

Comment:
let's keep naming convention
```suggestion
		tcType      string
		apiEndpoint string
		healthCheck bool
		err         error
```